packages/alfa-aria/src/feature.ts

```diff
@@ -71,7 +67,7 @@ export namespace Feature {
     /**
      * @internal
      */
-    readonly allowPresentational?: boolean;
+    readonly allowPresentational: boolean;
```

Comment:
I now realise that this has caused the public API to change in a way that makes it impossible to _not_ touch internal features from the outside as this internal property is now required 🙈